src/components/Header/Header.tsx

```diff
@@ -36,38 +54,66 @@ function Header() {
               </svg>
             </button>
           </div>
-          <ul
-            className={`${
-              isMenuOpen ? "block" : "hidden"
-            } md:flex space-x-6 mt-4 md:mt-0 `}
-          >
+          <ul className={`${isMenuOpen ? "block" : "hidden"} md:flex space-x-6 mt-4 md:mt-0`}>
             <li>
-              <Link
-                to="#"
-                className="text-gray-600 hover:text-gray-900 font-gilroy"
-              >
-                Events
-              </Link>
+              <a href="/" className="text-gray-600 hover:text-gray-900 font-gilroy">Home</a>
             </li>
             <li>
-              <Link
-                to="#"
-                className="text-gray-600 hover:text-gray-900 font-gilroy"
-              >
-                Community
-              </Link>
+              <a href="#" className="text-gray-600 hover:text-gray-900 font-gilroy">Communities</a>
             </li>
             <li>
-              <Link
-                to="/profile"
-                className="text-gray-600 hover:text-gray-900 font-gilroy"
-              >
-                Profile
-              </Link>
+              <a href="#" className="text-gray-600 hover:text-gray-900 font-gilroy">Events</a>
+            </li>
+            <li>
+              <a href="/" className="text-gray-600 hover:text-gray-900 font-gilroy">Skill Share</a>
+            </li>
+            <li className="relative">
+              <div onClick={toggleDropdown} className="flex items-center cursor-pointer">
+                <div className="h-10 w-10 bg-gray-300 rounded-full"></div>
+                <svg
```

Comment:
Noted. Since the design is not final and the svg was supposed to be there temporarily, I removed it, because I can see on Figma now that there is no drop down icon there any ways. So I just left the drop down functionality to be on the profile photo itself. When the design team give us the detailed design I will polish the icons part.